client/app/app.js

```diff
@@ -12,7 +12,8 @@ import ngOvhProxyRequest from '@ovh-ux/ng-ovh-proxy-request';
 import ngOvhUserPref from '@ovh-ux/ng-ovh-user-pref';
 import ngOvhWebUniverseComponents from '@ovh-ux/ng-ovh-web-universe-components';
 import ngTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
-import ngOvhUiRouterLineProgress from '@ovh-ux/ng-ui-router-line-progress';
```

Comment:
Seems not to be related to the addition of `@ovh-ux/ng-ui-router-layout` component.

A dedicated commit that tends to rename the import statement would facilitate the commit history view.